### ISSUE TITLE
Sidebat not displayed on tg page

### ISSUE THREAD

**Issue by anandr** [dokufreaks/plugin-tag#125]
When I create list of all tags with COUNT syntax and click on one of them sidebar is no longer displayed on that page. Can this be repaired?

**Comment by michitux** [comment]
Yes and no. There is no simple "fix" for this problem as this is the template which hides the sidebar whenever the action is not "show" and the tag plugin currently uses its own action. However the plugin could be changed to instead simply remplace the "page does not exist" page instead of using its own action. Actually when the page with the name of the tag exists that page is displayed already so it would only be logical to use the "show"-action also when the page does not exist.

**Comment by ryliejamesthomas** [comment]
>  If you change ($ACT == 'show') to ($ACT == 'show' || $ACT == 'showtag') this should work.

Can confirm that this works.
